.github/workflows/pipeline.yml

```diff
@@ -51,13 +51,23 @@ jobs:
         run: |
           sudo apt install $PREREQS
 
+      - name: Set up Go environment
```

Comment:
Why do we need golang support in our project?

Reply:
I have had multiple requests to provide a python package for our custom p4runtime.
A month or so ago, I received a request to provide a go package.
Kubernetes makes extensive use of go.
This PR supplies them.
We're working on generating a Python wheel as well. I'm in dependency hell getting it working right now.

Reply:
Can we add one of the K8's team members to review this so that we know we are aligned with the ask?